src/modules/tasks.js

```diff
@@ -0,0 +1,131 @@
+import _ from 'lodash';
+
+/* eslint-disable import/prefer-default-export */
+export class TaskList {
+  list = document.getElementById('list-items');
+
+  createEmpty = () => {
+    const tasks = [];
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  };
+
+  add = (e) => {
+    const taskName = document.getElementById('task').value;
+    const tasks = [];
+
+    const task = {
+      taskName,
+      completed: false,
+      index: tasks.length,
+    };
+    if (localStorage.getItem('tasks') === null) {
+      tasks.push(task);
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    } else {
+      const tasks = JSON.parse(localStorage.getItem('tasks'));
+      task.index = tasks.length;
+      tasks.push(task);
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    }
+    document.getElementById('form').reset();
+    e.preventDefault();
+  };
+
+  remove = (index) => {
+    const tasks = JSON.parse(localStorage.getItem('tasks'));
+    tasks.splice(index, 1);
+    for (let i = 0; i < tasks.length; i += 1) {
+      tasks[i].index = i + 1 - 1;
+    }
+
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+    window.location.reload();
+  };
+
+  edit = (index, description) => {
+    const tasks = JSON.parse(localStorage.getItem('tasks'));
+    for (let i = 0; i < tasks.length; i += 1) {
+      if (index === tasks[i].index) {
+        tasks[i].taskName = description;
+      }
+    }
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  };
+
+  removeComplete = () => {
+    const tasks = JSON.parse(localStorage.getItem('tasks'));
+    const task = tasks.filter((tsk) => tsk.completed === true);
```

Comment:
You can simply put tsk.completed instead of tsk.completed === true because tsk.completed is already a boolean value